css/styles.css

```diff
@@ -491,12 +494,12 @@ input[type = radio]:checked + label.label-radio .check {
    width: 12px;
    vertical-align: middle;
 }
-/* jQuery UI Combobox*/
-.pq-select-multiple {
-   display: inline-block;
-   width: 85% !important;
+tr[data-itemtype="PluginFormcreatorCondition"] .div_show_condition_logic {
+   visibility: hidden;
+}
+tr[data-itemtype="PluginFormcreatorCondition"] ~ tr[data-itemtype="PluginFormcreatorCondition"] .div_show_condition_logic {
+   visibility: visible;
```

Comment:
This hides the 1st boolean operator because it is meaningless, and got a report of user beiing disturbed about that.